recipes/geometric/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "geomeTRIC" %}
+{% set version = "0.8.8" %}
+
+# openssl sha256 filename
+{% set sha256 = "45ebea62a43045acd71d41516d732cb76bf65d760571a30454d4ff252dc1e10a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/leeping/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - numpy>=1.11
+    - python
+    - networkx
+    - six
+
+test:
+  imports:
+    - pytest
+    - geometric
+  requires:
+    - pytest
+  commands:
+    - py.test --pyargs geometric
+
+about:
+  home: http://github.com/leeping/geomeTRIC/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Molecular geometry optimizer in the TRIC coordinate system.'
+
+#  description: |
+#  doc_url: http://simplejson.readthedocs.io/
+  dev_url: http://github.com/leeping/geomeTRIC/
+
+extra:
+  recipe-maintainers:
+    - leeping
+    - dgasmith
+    - yudongqiu
```

Comment:
@lyudongqiu, are you ok with being a maintainer?

Reply:
@yudongqiu Please reply. (I saw a typo above)

Reply:
Yes.